packages/tools/devtools/devtools-example/src/App.tsx

```diff
@@ -401,65 +243,39 @@ function LoadingView(props: LoadingViewProps): React.ReactElement {
 /**
  * {@link AppView} input props.
  */
-interface AppViewProps extends ContainerInfo, HasContainerKey {}
+interface AppViewProps extends HasContainerKey {
+	/**
+	 * Container object.
+	 */
+	obj: AppData;
```

Comment:
Nit: How about `appData` instead of `obj`? It's a bit more descriptive.

Reply:
Renamed!